Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -1805,6 +1805,16 @@ bool RenderFlexibleBox::resolveFlexibleLengths(FlexSign flexSign, FlexItems& fle
     return !totalViolation;
 }
 
+inline ContentPosition resolveLeftRightAlignment(ContentPosition position, const RenderStyle& style, bool isReversed)
+{
+    if (position == ContentPosition::Left || position == ContentPosition::Right) {
```

Comment:
This function seems to indicate by the name and the first `if` guard here that passing in a `ContentPosition` that is not left/right is an error. Should we maybe also `ASSERT` here to indicate that and make sure that the caller only passes in valid values here?

Reply:
The function can handle any value. It only changes the left/right values.